packages/moode-player/build.sh

```diff
@@ -101,42 +102,43 @@ cd $BUILD_ROOT_DIR
 # ----------------------------------------------------------------------------
 
 # generate moode radio stations backup file (used for populating the station from the installer)
-cat $MOODE_DIR/var/local/www/db/moode-sqlite3.db.sql | sqlite3 $BUILD_ROOT_DIR/moode-sqlite3.db
-if [[ $? -gt 0 ]]
+
+if ! sqlite3 "${BUILD_ROOT_DIR}/moode-sqlite3.db" < "${MOODE_DIR}/var/local/www/db/moode-sqlite3.db.sql"
 then
     echo "${RED}Error: couldn't create temporary database!${NORMAL}"
     cd ..
     exit 1
 fi
 
-$MOODE_DIR/www/util/station_manager.py --db $BUILD_ROOT_DIR/moode-sqlite3.db --logopath $MOODE_DIR/var/local/www/imagesw/radio-logos --scope moode --export $BUILD_ROOT_DIR/moode-stations-full_$PKGVERSION.zip
-if [ ! -f $MAJOR_BASE_STATIONS ]
+"${MOODE_DIR}/www/util/station_manager.py" --db "${BUILD_ROOT_DIR}/moode-sqlite3.db" --logopath "${MOODE_DIR}/var/local/www/imagesw/radio-logos" --scope moode --export "${BUILD_ROOT_DIR}/moode-stations-full_$PKGVERSION.zip"
+if [ ! -f "../dist/binary/${MAJOR_BASE_STATIONS}" ]
```

Comment:
we have the full path in the if clause that is checking for its existence further down